generator/schemas/gen.go

```diff
@@ -207,21 +207,35 @@ func (g Generator) Generate(ctx *genall.GenerationContext) error {
 				return err
 			}
 
-			toAddMarkdownDescriptions := orderedmap.New()
-			json.Unmarshal(jsonSchema, toAddMarkdownDescriptions)
-			addMarkdownDescription(toAddMarkdownDescriptions)
-			schemaWithMarkdownDescriptions, err := json.MarshalIndent(toAddMarkdownDescriptions, "", "  ")
+			genutils.EditJSONSchema(
+				&currentJSONSchema,
+				func(schema *apiext.JSONSchemaProps) (newVisitor genutils.Visitor, stop bool) {
+					if schema != nil {
+						schema.Default = nil
+					}
+					return
+				})
+
+			(&currentJSONSchema).Title = (&currentJSONSchema).Title + " - IDE-targeted variant"
```

Comment:
Do you list somewhere what is the difference between normal and IDE-targeted variant?
Markdown, missing default, ...

Reply:
Not now. Would simply adding a `Readme.md` file in the `ide-targeted` folder be sufficient ?
And possibly explain the difference in the main description inside the Json schema ?

Reply:
> Does K8s/OS provision these defaults to CR with missing items?

Yes, but only with the `apiextensions/v1` CRD API: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting

Reply:
> Not now. Would simply adding a `Readme.md` file in the `ide-targeted` folder be sufficient ?
> And possibly explain the difference in the main description inside the Json schema ?

Done in commit https://github.com/devfile/api/pull/148/commits/6e6ac081df479eff476f4ee2f1de5a9585031a0f